test/fn/edit-contacts.spec.js

```diff
@@ -192,6 +194,139 @@ describe('edit-contacts', function() {
     } catch (err) {
       expect(err.message).to.be.equal('The column name(s) specified do not exist.');
     }
-  }); 
+  });
+
+  it('should load documents from provided directory if relevant argument is passed', async () => {
+    const db = sinon.stub(pouch, 'allDocs');
+    sinon.stub(
```

Comment:
better to break in every function when having more than one. Breaking in parameter is good when having many or too long.
Same with the other tests below
```js
sinon
  .stub(environment, 'extraArgs')
  .get(() => ['--columns=type', '--files=contact.type.csv', '--updateOfflineDocs=true', `--docDirectoryPath=${editedJsonDocs}`]);
```